test/python/sdpa/frost/test_sdpa_fwd_dsl_sm100.py

```
@@ -958,6 +958,105 @@ def _dense_buf(packed):
+    v_pk = torch.randn(T, H, d_v, device=dev, dtype=dtype)
+
+    def _stride(dd):
+        return (S_max * H * dd, dd, H * dd, 1)
+
+    def _poisoned_buf(packed, dd):
+        """Capacity-sized storage; only [0, T) is live, the tail is NaN."""
+        stor = torch.full((B * S_max * H * dd,), float("nan"), device=dev, dtype=dtype)
+        stor[: T * H * dd] = packed.reshape(-1)
+        return stor, stor.as_strided((B, H, S_max, dd), _stride(dd))
+
+    q_stor, q_gpu = _poisoned_buf(q_pk, d_qk)
```

Ruff (0.16.3)

[warning] Unpacked variable `q_stor` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)